### PR TITLE
Make Mangaflix more readable

### PR DESCRIPTION
**Pull request by ap29600**
added a few breaks on the longer lines.
The original script used the pattern 
```bash
LABEL=$(action)
```
to perform many actions. I find this hard to parse when compared to 
```sh
# LABEL
action
```
which also avoids spawning a whole new shell for each command.

I also replaced instances of `$(...)` to spawn subcommands with ``` `...` ```, which will work on `dash` and other posix compatible shells.

If your worry in writing this was minimalism, please note that this version is only 6 bytes larger than the original (2913 vs 2906)

### CODE REVIEW

**Comment by ap29600** [comment]
please note that I can't verify if this actually works the same as the original, I suspect manganato might be blocked in my region.

**Comment by thamognya** [comment]
Hey, I am adding this pull request as it works. Although there are a few conflicts, I will solve them and merge it.

**Comment by ThamognyaKodi** [comment]
Now it says I cant merge the branch with gui, but I decided to merge it manualy so I am going to have to close it without accepting it on github but accepting it on git.
